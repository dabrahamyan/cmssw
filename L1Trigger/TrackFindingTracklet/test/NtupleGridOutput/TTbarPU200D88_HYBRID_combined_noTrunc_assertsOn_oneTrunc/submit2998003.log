000 (2998003.000.000) 09/12 10:55:08 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_159281_1896>
...
040 (2998003.000.000) 09/12 12:45:48 Started transferring input files
	Transferring to host: <128.142.132.115:9618?addrs=128.142.132.115-9618+[2001-1458-301-5a--100-67]-9618&alias=b7g12p3756.cern.ch&noUDP&sock=slot1_4_5228_594e_161957>
...
040 (2998003.000.000) 09/12 12:45:48 Finished transferring input files
...
001 (2998003.000.000) 09/12 12:45:49 Job executing on host: <128.142.132.115:9618?addrs=128.142.132.115-9618+[2001-1458-301-5a--100-67]-9618&alias=b7g12p3756.cern.ch&noUDP&sock=startd_4551_2f99>
...
006 (2998003.000.000) 09/12 12:45:58 Image size of job updated: 72944
	38  -  MemoryUsage of job (MB)
	38276  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 12:50:59 Image size of job updated: 2524904
	1012  -  MemoryUsage of job (MB)
	1035956  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 12:55:59 Image size of job updated: 2533532
	1046  -  MemoryUsage of job (MB)
	1070512  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:00:59 Image size of job updated: 2533532
	1051  -  MemoryUsage of job (MB)
	1076032  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:06:00 Image size of job updated: 2697372
	1070  -  MemoryUsage of job (MB)
	1095068  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:11:01 Image size of job updated: 2697372
	1128  -  MemoryUsage of job (MB)
	1154880  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:16:01 Image size of job updated: 2697372
	1166  -  MemoryUsage of job (MB)
	1193376  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:26:03 Image size of job updated: 2697372
	1197  -  MemoryUsage of job (MB)
	1225452  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:31:03 Image size of job updated: 2697372
	1249  -  MemoryUsage of job (MB)
	1278436  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:36:04 Image size of job updated: 2893980
	1289  -  MemoryUsage of job (MB)
	1318952  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:41:05 Image size of job updated: 2893980
	1314  -  MemoryUsage of job (MB)
	1345480  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:46:05 Image size of job updated: 2893980
	1335  -  MemoryUsage of job (MB)
	1366624  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:51:06 Image size of job updated: 2893980
	1371  -  MemoryUsage of job (MB)
	1403348  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 13:56:07 Image size of job updated: 2893980
	1398  -  MemoryUsage of job (MB)
	1430580  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 14:01:07 Image size of job updated: 2904220
	1442  -  MemoryUsage of job (MB)
	1476264  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 14:06:07 Image size of job updated: 3133596
	1445  -  MemoryUsage of job (MB)
	1479220  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 14:16:08 Image size of job updated: 3133596
	1510  -  MemoryUsage of job (MB)
	1545692  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 14:21:08 Image size of job updated: 3133596
	1550  -  MemoryUsage of job (MB)
	1586572  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 14:26:09 Image size of job updated: 3133596
	1600  -  MemoryUsage of job (MB)
	1638336  -  ResidentSetSize of job (KB)
...
006 (2998003.000.000) 09/12 14:29:30 Image size of job updated: 7112044
	1600  -  MemoryUsage of job (MB)
	1638336  -  ResidentSetSize of job (KB)
...
040 (2998003.000.000) 09/12 14:29:30 Started transferring output files
...
040 (2998003.000.000) 09/12 14:29:30 Finished transferring output files
...
005 (2998003.000.000) 09/12 14:29:30 Job terminated.
	(1) Normal termination (return value 134)
		Usr 0 01:35:16, Sys 0 00:02:11  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:35:16, Sys 0 00:02:11  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	137809  -  Run Bytes Sent By Job
	702  -  Run Bytes Received By Job
	137809  -  Total Bytes Sent By Job
	702  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.97        1         1 
	   Disk (KB)            :   156           1   1613717 
	   Memory (MB)          :  1600        2000      2000 

	Job terminated of its own accord at 2023-09-12T12:29:30Z.
...
