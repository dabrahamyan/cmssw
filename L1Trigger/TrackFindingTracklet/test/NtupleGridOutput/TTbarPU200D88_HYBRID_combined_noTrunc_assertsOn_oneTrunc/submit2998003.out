using geometry D88 (tilted)
Tue Sep 12 14:29:09 CEST 2023
Thread 7 (Thread 0x2af42fdc2700 (LWP 184) "cmsRun"):
#0  0x00002af3fc29ab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002af3fc29abcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002af3fc29ac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002af40339a576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002af40339a639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002af3fc294ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002af3fc5a7b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2af42fbc1700 (LWP 183) "cmsRun"):
#0  0x00002af3fc29ab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002af3fc29abcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002af3fc29ac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002af40339a576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002af40339a639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002af3fc294ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002af3fc5a7b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2af42f9c0700 (LWP 182) "cmsRun"):
#0  0x00002af3fc29ab3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002af3fc29abcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002af3fc29ac6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002af40339a576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002af40339a639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002af3fc294ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002af3fc5a7b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2af42f7bf700 (LWP 181) "cmsRun"):
#0  0x00002af3fc29be9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002af4030d710f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002af4033082d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002af403308429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002af3fc294ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002af3fc5a7b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2af42f5be700 (LWP 180) "cmsRun"):
#0  0x00002af3fc5a80e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002af4030d15c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002af4030cdd4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002af4030d6818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002af3fc294ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002af3fc5a7b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2af41fd26700 (LWP 173) "cmsRun"):
#0  0x00002af3fc29c1d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002af4039f7ad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002af4039f86ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002af3fbe70f90 in std::execute_native_thread_routine (__p=0x2af40f521e30) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002af3fc294ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002af3fc5a7b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2af3fe62ae40 (LWP 155) "cmsRun"):
#0  0x00002af3fc59cddd in poll () from /lib64/libc.so.6
#1  0x00002af4039f7dcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002af4039f879c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002af4039fb24b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002af3fc4df387 in raise () from /lib64/libc.so.6
#6  0x00002af3fc4e0a78 in abort () from /lib64/libc.so.6
#7  0x00002af3fbe3c7dc in __gnu_cxx::__verbose_terminate_handler () at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#8  0x00002af3fbe479d6 in __cxxabiv1::__terminate (handler=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
#9  0x00002af3fbe46ab9 in __cxa_call_terminate (ue_header=ue_header@entry=0x2af43d29eee0) at ../../../../libstdc++-v3/libsupc++/eh_call.cc:54
#10 0x00002af3fbe473f1 in __cxxabiv1::__gxx_personality_v0 (version=<optimized out>, actions=6, exception_class=5138137972254386944, ue_header=0x2af43d29eee0, context=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_personality.cc:685
#11 0x00002af3fc28270f in _Unwind_RaiseException_Phase2 (exc=0x2af43d29eee0, context=0x7ffe15391570, frames_p=0x7ffe15391478) at ../../../libgcc/unwind.inc:64
#12 0x00002af3fc2830b6 in _Unwind_Resume (exc=0x2af43d29eee0) at ../../../libgcc/unwind.inc:241
#13 0x00002af40184e08e in throwStorageError(char const*, char const*, char const*, int) [clone .cold] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libUtilitiesStorageFactory.so
#14 0x00002af401861c31 in edm::storage::StorageAccountProxy::flush() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libUtilitiesStorageFactory.so
#15 0x00002af4041f2941 in TStorageFactoryFile::SysSync(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libIOPoolTFileAdaptor.so
#16 0x00002af3fa7f7aff in TFile::Flush() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libRIO.so
#17 0x00002af3fa803ef0 in TFile::WriteHeader() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libRIO.so
#18 0x00002af3fa803bf9 in TFile::Write(char const*, int, int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libRIO.so
#19 0x00002af405f45236 in TFileService::~TFileService() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libCommonToolsUtilAlgos.so
#20 0x00002af405f2c6d9 in edm::serviceregistry::ServiceWrapper<TFileService>::~ServiceWrapper() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginCommonToolsToolsUtilAlgos_plugins.so
#21 0x00002af3f9a91bae in edm::serviceregistry::ServicesManager::~ServicesManager() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreServiceRegistry.so
#22 0x00002af3f9a91cdf in edm::serviceregistry::ServicesManager::~ServicesManager() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreServiceRegistry.so
#23 0x00002af3f9a91cdf in edm::serviceregistry::ServicesManager::~ServicesManager() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreServiceRegistry.so
#24 0x00002af3f9d26e77 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#25 0x00002af3f9d53227 in edm::EventProcessor::~EventProcessor() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#26 0x0000000000409b11 in (anonymous namespace)::EventProcessorWithSentry::~EventProcessorWithSentry() ()
#27 0x0000000000409739 in main ()
